tensorboard/plugins/custom_scalar/tf_custom_scalar_dashboard/tf-custom-scalar-margin-chart-card.html

```diff
@@ -719,12 +722,14 @@
         );
       },
       _matchListEntryColorUpdated(event) {
-        const [maybeDomRepeat] = event.path;
-        if (maybeDomRepeat.nodeName !== 'DOM-REPEAT') return;
+        const domRepeat = this.$$('#match-list-repeat');
```

Comment:
Curious why the change from `event.path` to this selector.  Is .path, .target not reliable?

Reply:
We only needed `event.path` because `dom-if` also emits the same event and we wanted to discern the events. Now that we have more localized listener, we simply don't need one.

Reply:
The `event.path` works before and after, just curious whether we gain something by switching to a querySelector.  Either way is fine with me, though